pom.xml

```diff
@@ -338,12 +338,18 @@
                            <includes>
                               <include>bootstrap/dist/js/bootstrap.min.js</include>
                               <include>bootswatch/dist/*/bootstrap.min.css</include>
-                              <include>font-awesome/css/font-awesome.min.css</include>
-                              <include>font-awesome/fonts/*.*</include>
                               <include>html5shiv/dist/html5shiv.min.js</include>
                               <include>jquery/dist/jquery.min.js</include>
                            </includes>
                         </resource>
+                        <resource>
+                           <directory>node_modules/@fortawesome/fontawesome-free</directory>
+                           <targetPath>font-awesome</targetPath>
+                           <includes>
+                              <include>css/all.min.css</include>
+                              <include>webfonts/*.*</include>
+                           </includes>
+                        </resource>
```

Comment:
With the move to 5, Font Awesome changed their npm location. They now have fontawesome-free in the public repo and a private repo for the paid version. I split the resources out so they could still sit under the `/lib/font-awesome/..` URI.